.github/workflows/yocto-build-deploy.yml

```diff
@@ -585,6 +595,9 @@ jobs:
 
           cat "${AUTO_CONF_FILE}"
 
+          >&2 eval "$(ssh-agent)"
+          >&2 echo "${YOCTO_SSH_PRIVATE_KEY_B64}" | base64 -d | ssh-add -
```

Comment:
```suggestion
          >&2 echo "${{ secrets.YOCTO_SSH_PRIVATE_KEY_B64 }}" | base64 -d | ssh-add -
```
This way we can avoid the secret being exposed to the build container as an env var, and limit the scope a bit.

Reply:
Testing right now.